pkg/storaged/vdo-details.jsx

```diff
@@ -171,7 +171,7 @@ export class VDODetails extends React.Component {
             }
 
             function wipe_with_teardown(block) {
-                return block.Format("empty", { 'tear-down': { t: 'b', v: true } });
+                return block.Format("empty", { 'tear-down': { t: 'b', v: true } }).then(reload_systemd);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-131055-4172823d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)